plugin/src/main/groovy/gorm/tools/repository/GormRepo.groovy

```diff
@@ -17,6 +17,7 @@ import org.grails.datastore.gorm.GormEntity
 import org.grails.datastore.gorm.GormInstanceApi
 import org.grails.datastore.gorm.GormStaticApi
 import org.grails.datastore.mapping.core.Datastore
+import org.hibernate.ObjectNotFoundException
```

Comment:
you keep trying to add hibernate dependecies here. no hibernate dependencies unless its under gorm.tools.hibernate.